### ISSUE TITLE
locationFromAddress using in web

### ISSUE THREAD

**Issue by paulobreim** [Baseflow/flutter-geocoding#242]
### Please check the following before submitting a new issue.

- [X] I have searched the [existing issues](https://github.com/baseflow/flutter-geocoding/issues).
- [X] I have carefully [read the documentation](https://github.com/baseflow/flutter-geocoding/blob/main/geocoding/README.md).

### Please select for which platform(s) you need help

- [X] Android
- [ ] iOS

### Your question

I have a program that is working very well on Android, but when I run it on the web/wasm, it never returns from the locationFromAddress call, not even due to timeout.
I would like to know if there is a problem with this.

```

placeMarkAdd = await locationFromAddress(
             '$typedaddress  $typednumber, São Paulo')
         .timeout(const Duration(seconds: 10))
          ignore: body_might_complete_normally_catch_error
         .catchError((error) async {
       if (error is TimeoutException) {
         debugPrint("timeout gps");
       } else {
         debugPrint("error $error");
       }
     });
```

---
Exception has occurred.
Error
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 296:3       throw_
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 586:18  nullCheck
packages/geocoding/geocoding.dart 16:33                                           locationFromAddress
packages/imovelweb/page1.dart 251:15                                              pegaLatLon
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 84:54                runBody
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 127:5                _async
packages/imovelweb/page1.dart 240:26                                              pegaLatLon
packages/imovelweb/page1.dart 563:15                                              pegaIptu
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 45:50                <fn>
dart-sdk/lib/async/zone.dart 1661:54                                              runUnary
dart-sdk/lib/async/future_impl.dart 163:18                                        handleValue
dart-sdk/lib/async/future_impl.dart 847:44                                        handleValueCallback
dart-sdk/lib/async/future_impl.dart 876:13                                        _propagateToListeners
dart-sdk/lib/async/future_impl.dart 652:5                                         [_completeWithValue]
dart-sdk/lib/async/future_impl.dart 722:7                                         callback
dart-sdk/lib/async/schedule_microtask.dart 40:11                                  _microtaskLoop
dart-sdk/lib/async/schedule_microtask.dart 49:5                                   _startMicrotaskLoop
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 181:7                <fn>
----

tks
paulo

### Version

3.0.0

**Comment by TimHoogstrate** [comment]
Dear @paulobreim,

Can you elaborate a bit on the subject? How do you run this on web? On Android in a browser? And what browser?

Kind regards,

**Comment by paulobreim** [comment]
> Dear @paulobreim,
> 
> Can you elaborate a bit on the subject? How do you run this on web? On Android in a browser? And what browser?
> 
> Kind regards,

Dear @TimHoogstrate ,

Thank you for your answer.

I have an app that has been in the store for some time now and it works perfectly. (https://play.google.com/store/apps/details?id=cbp.net.imoveissp)

It is an app that has all the properties in the city of São Paulo. When the user types the address, I check the syntax of the street name and the geographic location. So far, everything works very well.

I decided to create a version for web/wasm but I found a lot of incompatibility issues and so I started to generate a new version for web, step by step.
I am using vscode and when I test it, I test it in the chrome option and use the debugging feature to see if everything is going well.
When the program calls the await locationFromAddress instruction, it stops waiting for the return, which never comes back.|
I can do other tests if necessary.

Regards
Paulo
